packages/moshicam/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@rabbitholegg/questdk-plugin-moshicam",
+  "private": true,
+  "version": "1.0.0-alpha.0",
+  "exports": {
+    "require": "./dist/cjs/index.js",
+    "import": "./dist/esm/index.js",
+    "types": "./dist/types/index.d.ts"
+  },
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "description": "Plugin for moshicam",
+  "scripts": {
+    "bench": "vitest bench",
+    "bench:ci": "CI=true vitest bench",
+    "build": "pnpm build:types && pnpm build:esm && pnpm postbuild:esm && pnpm build:cjs && pnpm postbuild:cjs && BABEL_ENV=esmBundled pnpm rollup -c",
```

Comment:
We are using vite now for the build process so you will need to update the package.json and create a vite config file.

You can copy these files from another package and update the name.

You can see what needs to be changed here: https://github.com/rabbitholegg/questdk-plugins/pull/429/files